### PR TITLE
PIM-5363 : Import/export Job management

### PR DESCRIPTION
**Pull request by arnolanglade**
| Q | A |
| --- | --- |
| Bug fix? | y |
| New feature? | n |
| BC breaks? | n |
| Doc PR | n |

The `Akeneo Native Connector` has been removed last week,  the new connector used by this bundle is `Akeneo CSV connector`

### CODE REVIEW

**Comment by pierallard** [comment]
GTM for the code. Ask @samir before merge to check data.

File: Resources/config/internal_jobs.yml [hunk 1]
```diff
@@ -135,3 +135,88 @@ jobs:
...
             filePath: /tmp/attribute-option.csv
+    xlsx_product_quick_export:
+        connector: Akeneo Mass Edit Connector
+        alias: xlsx_product_quick_export
+        label: XLSX product quick export
+        type: quick_export
+        configuration:
+            withHeader: true
+            filePath:   /tmp/products_export_%locale%_%scope%_%datetime%.xlsx
+    xlsx_category_import:
```
Comment (SamirBoulil) [inline Resources/config/internal_jobs.yml:146]: Does not exist yet but I merge it tomorow

File: Resources/config/internal_jobs.yml [hunk 2]
```diff
@@ -135,3 +135,88 @@ jobs:
...
+            circularRefsChecked: true
+    xlsx_category_export:
+        connector: Akeneo XLSX Connector
+        alias:     xlsx_category_export
+        label:     Category export
+        type:      export
+        configuration:
+            withHeader: true
+            filePath: /tmp/category.xlsx
+    xlsx_association_type_import:
```
Comment (SamirBoulil) [inline Resources/config/internal_jobs.yml:162]: Same here :)

**Comment by SamirBoulil** [comment]
Seems good to me. Did you try to import the generated files ?

**Comment by pierallard** [comment]
We have to close https://github.com/akeneo-labs/DataGeneratorBundle/issues/29 when merged ;)

**Comment by solivier** [comment]
gtm if Samir's card is merged

**Comment by arnolanglade** [comment]
@solivier yes it is!

File: Resources/examples/fixtures_enterprise.yml [hunk 3]
```diff
@@ -224,6 +224,44 @@ data_generator:
...
+                    enclosure:          '"'
+                    escape:             '\'
+                    enabled:            true
+                    categoriesColumn:   categories
+                    familyColumn:       family
+                    groupsColumn:       groups
+                    realTimeVersioning: true
+                    decimalSeparator:   .
+                    dateFormat: yyyy-MM-dd
+            xlsx_clothing_product_import_with_rules:
```
Comment (skeleton) [inline Resources/examples/fixtures_enterprise.yml]: clothing ?

Comment (jjanvier) [inline Resources/examples/fixtures_enterprise.yml]: clothing?

File: composer.json [hunk 4]
```diff
@@ -25,15 +25,13 @@
     "require": {
         "php": ">=5.4.4",
         "akeneo/pim-community-dev": "dev-master@dev",
-        "fzaninotto/faker": "1.4.0",
-        "box/spout": "^2.4"
```
Comment (jjanvier) [inline composer.json:29]: is it normal? we don't use spout here?

**Comment by jjanvier** [comment]
gtm once fixed

File: composer.json (hunk 4 above)
Comment (arnolanglade) [inline composer.json:29]: We don't use it!

File: Resources/examples/fixtures_enterprise.yml (hunk 3 above)
Comment (arnolanglade) [inline Resources/examples/fixtures_enterprise.yml]: copy/paste :cry:

**Pull request merged**